migration/sql-files/062-workitem-related-changes.sql

```diff
@@ -0,0 +1,77 @@
+-- add a column to record the timestamp of the latest addition/change/removal of a comment on a workitem
+ALTER TABLE work_items ADD COLUMN commented_at timestamp with time zone;
+
+-- trigger to fill the `commented_at` column when a comment is added, updated or removed (soft delete, it's a record update)
+CREATE FUNCTION workitem_comment_insert_timestamp() RETURNS trigger AS $$
```

Comment:
Please put the comment inside the `CREATE FUNCTION` then it will be shown when you're on the `psql` command line interface.